config/initializers/geocoder.rb

```diff
@@ -15,3 +15,16 @@
   Geocoder.configure(:lookup => :nominatim)
 end
 
+Geocoder::Lookup::Test.add_stub(
```

Comment:
Could you please move this to `config/environments/test.rb` along with the other Geocoder stubs?